mergin/client.py

```diff
@@ -821,14 +828,16 @@ def project_user_permissions(self, project_path):
         :param project_path: project full name (<namespace>/<name>)
         :return dict("owners": list(usernames),
                      "writers": list(usernames),
+                     "editors": list(usernames),
                      "readers": list(usernames))
         """
         project_info = self.project_info(project_path)
         access = project_info.get("access")
         result = {}
-        result["owners"] = access.get("ownersnames")
-        result["writers"] = access.get("writersnames")
-        result["readers"] = access.get("readersnames")
+        result["owners"] = access.get("ownersnames", [])
+        result["writers"] = access.get("writersnames", [])
+        result["editors"] = access.get("editorsnames", [])
```

Comment:
I would suggest to add `editors` key to the result only if the server is new enough... and also add to docstring that `editors` is not present for older servers